compiler/q-implant/include/QImplant.h

```diff
@@ -28,9 +28,11 @@ class QImplant final
 {
 public:
   QImplant(const std::string &path) : _path(path) {}
-
```

Comment:
please move this format change in another PR.

Reply:
Q) why remove this line? personally this helps reading the code and looks better.

Reply:
> please move this format change in another PR.

1) Does it means that add function signature in .h in another PR, and add implementation in this PR? (Maybe rebase this branch to merged one?)
2) Or, just `format change` means that only line removal?

> Q) why remove this line? personally this helps reading the code and looks better.

And I also thought that line separation looks better too.
I just removed the line by thinking about there is convention that I don't know.

> plz put this like
```
class QImplant final
{
public:
  QImplant(const std::string &path) : _path(path) {}
  void write(loco::Graph *g);

private:
  void forward_qparam(loco::Graph *g);

private:
  const std::string &_path;
```

I will keep the line after check what I have to do.
When case 1, I will separate `.h` first, and change this PR.
Or when case 2, I will just keep the line in this PR.

Thank You!

Reply:
> Or, just format change means that only line removal?

this one.

Reply:
Add the line as before.

Thank You!